SmoothBankCard/src/main/java/com/ss/sbank/controller/cards/CardController.java

```diff
@@ -1,24 +1,63 @@
 package com.ss.sbank.controller.cards;
 
 import java.util.List;
+import java.util.Map;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.ss.sbank.de.cards.Card;
+import com.ss.sbank.de.cards.CardType;
 import com.ss.sbank.service.cards.CardService;
+import com.ss.sbank.service.cards.CardTypeService;
 
 @RestController
-@RequestMapping("/cards")
+@CrossOrigin
+@RequestMapping("/api/cards")
 public class CardController {
-	
+
 	@Autowired
 	private CardService service;
+
+	@Autowired
+	private CardTypeService ctService;
 	
-	@RequestMapping("/getAll")
-	public List<Card> getAllCards(){
+	@GetMapping("/TEST-PLATFORM")
+	public List<Card> getAll(){
+		/**
+		 * Now normally I wouldn't make this method do anything because it is a major security problem, but for
+		 * the sake of demonstrating the program it will remain.
+		 * #########REMOVE THIS METHOD ENTIRELY BEFORE DEPLOYING ANYWHERE##############
+		 */
 		return service.getAllCards();
```

Comment:
The controller method return type should be ResponseEntity

Reply:
that one isn't returning a response entity because it's not intended to remain in the code. Since this is the user portion we don't want to leave it in there, hence the comment to remove the method. It'll be destroyed before being submitted and was only there for debugging reasons.